contrib/friendz/src/friendz_android.nit

```diff
@@ -21,11 +22,20 @@ redef class App
 	redef fun window_created
 	do
 		super
-
```

Comment:
Did you have any reason to remove the last two empty lines? They looked like they were appropriate.